build.sbt

```diff
@@ -650,17 +584,19 @@ lazy val gradleBloop212 = project
     ) / "gradle-bloop" / "target" / "gradle-bloop-2.12").getAbsoluteFile
   )
   .settings(
-    libraryDependencies += Dependencies.classgraph % Test,
-    publishLocal := publishLocal.dependsOn((jsonConfig212.jvm / publishLocal)).value
+    libraryDependencies ++= Seq(
+      Dependencies.classgraph % Test,
+      Dependencies.bloopConfig
+    )
   )
+  .dependsOn(integrationUtils212 % "test->compile")
```

Comment:
It's kind of unrelated to this PR but why not put all the settings together in a single `.settings(...)` and all the internal dependencies in a single `.dependsOn(...)`.

I find it misleading that there are two `.dependsOn` on the same project, and spread over many `.setttings`. It is easy to miss one.

Reply:
Ah I actually didn't realize there was another `dependsOn` up above. I'll move this.

In general, I totally agree with you. I have a couple small things on my list that I hit on while working on this and cleaning up some of this stuff is one of them. However I want to do it in a separate pr.

Reply:
:+1: